books/RayTracingTheNextWeek.html

```diff
@@ -130,9 +154,9 @@
 
 Adding Moving Spheres
 ----------------------
-We also need a moving object. I’ll create a sphere class that has its center move linearly from
-`center0` at `time_start` to `center1` at `time_end`. Outside that time interval it continues on, so
-those times need not match up with the camera aperture open and close.
+Now to create a moving object. I’ll create a sphere class that has its center move linearly from
+`center0` at time 0 to `center1` at time 1. (It continues on outside that time interval, so it
```

Comment:
Consider:

time 0 -> start time
time 1 -> end time

Reply:
We need to really make clear the implicit time values of 0 and 1, since we're coding to that. For clarify, I've replaced "time 0" and "time 1" with "time=0" and "time=1".